components/dialog/test/dialog.visual-diff.js

```diff
@@ -48,19 +48,6 @@ describe('d2l-dialog', () => {
 						await visualDiff.screenshotAndCompare(page, this.test.fullTitle());
 					});
 
-					it('scroll bottom shadow', async function() {
```

Comment:
These tests did not need to be repeated for both wide & narrow cases.  Trying to optimize.